src/js/tech/ChromecastTech.js

```diff
@@ -535,6 +637,20 @@ ChromecastTech = {
       this._eventListeners.push(listener);
    },
 
+   /**
+    * on dispose
```

Comment:
please either rewrite either as a sentence, or if there's nothing to write (as in some other private methods), use the method name ... no particular preference here, but want to maintain consistency with the surrounding code

Reply:
https://github.com/silvermine/videojs-chromecast/pull/112/commits/6cae9ea755871a02e2518e557fd26c9549e1fea6